starters/gatsby/src/pages/products/{NacelleProduct.content__handle}.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from 'react';
```

Comment:
This is really cool, I had no idea you could do this with the file name. https://www.gatsbyjs.com/docs/reference/routing/file-system-route-api/#field-syntax

Reply:
Yeah! At first I wasn't crazy about the syntax, but I really appreciate how flexible it is, especially when it comes to nested fields.

Reply:
This is super cool